pkg/kubelet/kubelet_volumes.go

```diff
@@ -125,9 +125,9 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(pods []*v1.Pod, runningPods []*kubecon
 
 	logSpew := func(errs []error) {
 		if len(errs) > 0 {
-			glog.Errorf("%v : There were a total of %v errors similar to this. Turn up verbosity to see them.", errs[0], len(errs))
+			glog.Errorf("%v : There were a total of %d errors similar to this. Turn up verbosity to see them.", errs[0], len(errs))
 			for _, err := range errs {
-				glog.V(5).Infof("Orphan pod: %v", err)
+				glog.V(3).Infof("Orphan pod: %v", err)
```

Comment:
What's the verbosity in most Kubernetes deployments? Don't want to bloat everyone's logs all of a sudden.

Reply:
yes, you are right, I just want to unify the log level with https://github.com/zhangxiaoyu-zidif/kubernetes/blob/afddd7bff41e26b90680510dca64d73ce7bd8e12/pkg/kubelet/kubelet_volumes.go#L106.
I suppose that they should be at same log level.